package.json

```
@@ -88,7 +88,7 @@
 	},
 	"lint-staged": {
 		"*": "prettier --write",
-		"*.js": "eslint",
+		"*.js": "eslint --fix",
 		"*.css": "stylelint"
```

Stylelint can/should fix too:

```suggestion
		"*.css": "stylelint --fix"
```

---

Okay, I've just changed it :slightly_smiling_face: